app/cypress/e2e/cif/project-revision/create-first-project-revision.cy.js

```diff
@@ -167,19 +174,34 @@ describe("when creating a project, the project page", () => {
 
     // funding agreement section
     cy.checkFundingAgreementForm(
+      true,
       "$222.00",
       "60.00 %",
-      "$333.00",
       "$800.00",
+      "70.60%",
       /Jan(\.)? 1, 2020/,
       /Feb(\.)? 2, 2020/,
+      "$333.00",
       "$1,133.00",
-      true,
       "20 %"
     );
     // additional funding sources section
     cy.findByText(/Additional Funding Source 1/i).should("be.visible");
 
+    // payment tracker section
```

Comment:
I think it's important to do assertions for calculated values because otherwise the only e2e check we have for them is happo screenshots